mergePDF/pdf_merger.py

```diff
@@ -0,0 +1,19 @@
+from PyPDF2 import PdfFileMerger, PdfFileReader
+from os import listdir
+from os.path import isfile, join
+import os
+
+path = input("Path to pdf files: ")        # "/Users/huygiang/Desktop/songs/Easter-sunday/"
+# print(os.listdir(path))
+
+pdf_files = [f for f in os.listdir(path) if isfile(join(path, f))]
+# print(pdf_files)
+
+filename = input("Name it: ")
```

Comment:
I was a little confused as to where the output file was stored. Definitely won't be storing back to source directory!